tensorboard/components/vz_chart_helpers/plottable-interactions.js

```diff
@@ -200,6 +200,15 @@ var vz_chart_helpers;
     }
   }
 
+  Plottable.Interaction.prototype._isInsideComponent = function(p) {
+    return (
+      0 <= p.x &&
+      0 <= p.y &&
+      p.x < this._componentAttachedTo.width() &&
+      p.y < this._componentAttachedTo.height()
+    );
+  };
+
```

Comment:
revert?

Reply:
Yup, thanks. I forgot which branch I was in :\